vol_test.c

```diff
@@ -183,6 +187,10 @@ main(int argc, char **argv)
         }
     }
 
+    if (NULL == (test_path_prefix = getenv("HSDS_PATH"))) {
```

Comment:
Since this prefix will probably be used by other connectors at some point, we should have a more generic name for the environment variable. Maybe HDF5_VOL_TEST_PATH_PREFIX?